### PR TITLE
New Marketing page

### PR DESCRIPTION
**Pull request by Narek-Avdalyan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Marketing</title>
+    <link
+      rel="stylesheet"
+      href="http://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script
+      src="https://kit.fontawesome.com/a076d05399.js"
```
Comment (SergeyGhazaryan) [inline index.html:15]: please change this to svg

File: index.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (SergeyGhazaryan) [inline index.html:6]: please add favicon

File: index.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+              <div style="position: relative">
+                <img src="images/sound.jpg" alt="" />
+                <div class="anim-cont-3"><span>Blorim pipsum</span></div>
+              </div>
+            </div>
+
+            <div class="carusel-footer">
+              <i class="previous-img fa fa-arrow-circle-left"></i>
+              <div>
+                <button class="carusel-button button-1"></button>
```
Comment (SergeyGhazaryan) [inline index.html:56]: please check all clasnames

File: index.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    anim3.style.animation = "blorim 2s";
+  }
+};
+
+document.querySelector(".next-img").addEventListener("click", function () {
+  offset += 100;
+  if (offset > 200) {
+    offset = 0;
+  }
+  buttonbackgraund(offset);
```
Comment (SergeyGhazaryan) [inline index.js:43]: replace to buttonBackground

File: index.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
+let offset = 0;
+const slider = document.querySelector(".slider");
+const but1 = document.querySelector(".button-1");
+const but2 = document.querySelector(".button-2");
+const but3 = document.querySelector(".button-3");
+const anim1 = document.querySelector(".animation-content");
+const anim2 = document.querySelector(".anim-cont-2");
+const anim3 = document.querySelector(".anim-cont-3");
```
Comment (SergeyGhazaryan) [inline index.js:8]: please again check all names

File: index.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+const buttonbackgraund = (offset) => {
```
Comment (SergeyGhazaryan) [inline index.js:10]: please change this garbage logic

File: index.js [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+      ButtonProperetis.animations.style.animation = "none";
+    }
+  }
+};
+
+document.querySelector(".next-img").addEventListener("click", function () {
+  offset += 100;
+  if (offset > 200) {
+    offset = 0;
+  }
```
Comment (SergeyGhazaryan) [inline index.js:48]: why did you use this logic?

File: index.js [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+  buttonBackgraund(offset);
+});
+
+document.querySelector(".previous-img").addEventListener("click", function () {
+  offset -= 100;
+  if (offset < 0) {
+    offset = 200;
+  }
```
Comment (SergeyGhazaryan) [inline index.js:56]: why did you use this logic?

File: index.js [hunk 9]
```diff
@@ -0,0 +1,68 @@
+let offset = 0;
+const slider = document.querySelector(".slider");
+const firstSlide = document.querySelector(".first-slide");
+const secondSlide = document.querySelector(".second-slide");
+const thirdSlide = document.querySelector(".third-slide");
+
+const changeImageButton = [
+  (firstImage = {
```
Comment (Shitikyan) [inline index.js:8]: why this is needed `firstImage` `secondImage` and so on ?

File: index.js [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+  }),
+  (thirdImage = {
+    button: thirdSlide,
+    animations: document.querySelector(".third-span-animation"),
+    animationName: "blorim 2s",
+  }),
+];
+
+const buttonBackground = (offset) => {
+  slider.style.right = offset + "%";
```
Comment (Shitikyan) [inline index.js:26]: `offset` we have also in global, better to use another name

File: index.js [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+      changeImageButton[i].animations.style.animation =
+        changeImageButton[i].animationName;
+    } else {
+      changeImageButton[i].button.style.backgroundColor = "inherit";
+      changeImageButton[i].animations.style.animation = "none";
+    }
+  }
+};
+
+document.querySelector(".next-img").addEventListener("click", function () {
```
Comment (Shitikyan) [inline index.js:39]: use arrow functions?

File: index.js [hunk 12]
```diff
@@ -0,0 +1,68 @@
...
+  offset += 100;
```
Comment (Shitikyan) [inline index.js:40]: use constants instead of magic numbers

**Pull request merged**